Makefile

```diff
@@ -1,3 +1,136 @@
+# Used for Map Matching
+MAP_MATCHING_COORDINATES = 13.418946862220764,52.50055852688439;13.419011235237122,52.50113000479732;13.419756889343262,52.50171780290061;13.419885635375975,52.50237416816131;13.420631289482117,52.50294888790448
+
+checkstyle:
```

Comment:
lines 4 - 133 are duplicate of make commands already in the file, they can be removed.